TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -724,18 +724,22 @@ private static SymbolReference ResolveProcedureName(SymbolTable table, SymbolRef
                 DiagnosticUtils.AddError(node, nodeTypeName + " \'" + node.Name + "\' already declared");
 
             // a section/paragraph is empty when it has no child or when its child/children is/are an {End} node
-            bool empty = true;  // default value
-            foreach (Node sentence in node.Children)
+            if (!node.Name.Equals("INIT-LIBRARY", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
@LANOYDO, @smedilol @fm-117 
So this is a restriction, as soon as  one of  the Section or a Paragraph is named INIT-LIBRARY this check is no longer performed?
I say that because I know that INIT-LIBRARY paragraph name is checked by the LibraryChecker, maybe we can identify that the paragraph is the INIT-LIBRARY paragraph and only ignore the check in this situation.
This information can be set by the LibraryChecker by adding for instance a IsInitLibrary boolean property to the Paragraph, but in this case we must assume that the LibraryChecker is called before the SectionOrParagraphUsageChecker.

Reply:
We can remove this test.
An `exit` statement can be put in paragraph INIT-LIBRARY to avoid the warning.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1479/commits/01c239da31a567402e43d3e2e6c655c577177b67